### PR TITLE
Companies mentions and love

### PR DESCRIPTION
**Pull request by CaroFG**

### CODE REVIEW

**Comment by CaroFG** [comment]
Not sure where to put the links to those files, in the assets readme or in the main readme?

File: assets/companies.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Companies using MeiliSearch
```
Comment (fharper) [inline assets/companies.md:1]: Compagnies & Organizations?

File: assets/companies.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+| Companies | Description |How they use us |Source  |
+|--|--|--|--|
+| [Platform.sh](https://docs.platform.sh/) | Platform-as-a-Service built especially for continuous deployment |	Cross-site search  | [tweet](https://twitter.com/robertDouglass/status/1262395403363921922) |
```
Comment (fharper) [inline assets/companies.md:5]: Can you order ASC by companies please

File: assets/meili-love.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Love from our users
```
Comment (fharper) [inline assets/meili-love.md:1]: Can you put the text of the tweets, and date please (link the text to the link you added here). Order them by date ASC please :D

File: assets/mentions.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Mentions of MeiliSearch
```
Comment (fharper) [inline assets/mentions.md:1]: Put the date, order them by ASC date.

Link to the date so links are hidden instead of a list of links.

**Review by fharper — COMMENTED** [review]
> Not sure where to put the links to those files, in the assets readme or in the main readme?

In the asset README.

**Comment by CaroFG** [comment]
Fixes #204

File: assets/meili-love.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+- 20/03/2020: [love the performance and simplicity of MeiliSearch](https://github.com/meilisearch/MeiliSearch/issues/535#issue-585176606)
```
Comment (fharper) [inline assets/meili-love.md:3]: My bad, I should have mention it, but can you put the date YYYY-MM-DD :D

It make more sense when you order by date ASC 😬

File: assets/mentions.md (hunk 4 above)
Comment (fharper) [reply assets/mentions.md:1]: Same for date here 😬

**Review by fharper — CHANGES_REQUESTED** [review]

**Pull request merged**